### PR TITLE
Add security guidelines

### PR DESCRIPTION
**Pull request by terhorstd**
This file defines the NEST security infrastructure. As discussed in the NEST-Initative Board.

The information will be available also in the project's [Security Advisories](https://github.com/nest/nest-simulator/security/policy) page on GitHib (see next to Wiki above)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+# Security Policy
+
+The NEST-Simulator is a research oriented software and as such is expected to
```
Comment (jougs) [inline SECURITY.md]:
```suggestion
The NEST simulator is research oriented software and as such is expected to
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+run mainly in well protected environments. In case it is found that this
+software can be used to violate security mechanisms, developers will try to
+provide patches that mitigate the risk of this misuse.
+
+The current policy is SECURITY.md in nest/nest-simulator master branch, which
```
Comment (jougs) [inline SECURITY.md]:
```suggestion
The current policy is explained in SECURITY.md in the nest-simulator master branch.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+can also be found at <https://github.com/nest/nest-simulator/security/policy>.
```
Comment (jougs) [inline SECURITY.md]:
```suggestion
The policy can also be found at <https://github.com/nest/nest-simulator/security/policy>.
```

Should we also (automagically) mirror a rendered version of the SECURITY.md file to somewhere on the nest-simulator.org homepage?

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Supported Versions
+
+Generally only the last version will receive security related updates. In severe
```
Comment (jougs) [inline SECURITY.md]:
```suggestion
Generally only the latest version will receive security related updates. In severe
```

**Comment by Silmathoron** [comment]
Out of curiosity, what kind of security issues are we talking about here?

**Comment by terhorstd** [comment]
I think many things can be considered security related bugs. Basically anything that raises concerns of the user or machine/network admins. Main point is to define a way of contacting the developers on a secure channel. If a user would find a way to do any mischief with the simulator, it wouldn't be a good idea to put that into normal issue tracking or onto the mailing list. This of course assumes (a) something like this is possible, however unlikely, and (b) that it is a person is not tying to be next years supervillain.

**Pull request merged**